src/components/symptom-shark/container/ReportBuilder/ReportBuilder.tsx

```diff
@@ -0,0 +1,221 @@
+import React, { useEffect, useRef, useState } from "react";
+import { Button, DateRangeContext, DateRangeNavigator, LoadingIndicator, Section, TrackerItem, UnstyledButton } from "../../../presentational";
+import language from "../../../../helpers/language";
+import MonthReport from "../../presentational/MonthReport/MonthReport";
+import { DailyLogEntry, SymptomConfiguration, SymptomSharkConfiguration, TreatmentConfiguration } from "../../../..";
+import { add } from "date-fns";
+import symptomSharkData from "../../helpers/symptom-shark-data";
+import { demoLogEntries, demoSymptoms, demoTreatments } from "../../helpers/demo-data";
+import MyDataHelps from "@careevolution/mydatahelps-js";
+import renderPdf from "../../../../helpers/renderPdf";
+import { SymptomSharkVisualizationContext } from "../VisualizationCoordinator/VisualizationCoordinator";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faCheckCircle, faCircle } from "@fortawesome/free-regular-svg-icons";
+import { faChevronDown, faChevronUp } from "@fortawesome/free-solid-svg-icons";
+import "./ReportBuilder.css"
+
+export interface SymptomSharkReportBuilderProps {
+    productLogo?: string;
+    productName?: string;
+    productUrl?: string;
+    previewState?: "default";
+    innerRef?: React.Ref<HTMLDivElement>;
+}
+
+export default function ReportBuilder(props: SymptomSharkReportBuilderProps) {
+    var currentDate = new Date();
+    currentDate = new Date(currentDate.getFullYear(), currentDate.getMonth(), currentDate.getDate(), 0, 0, 0, 0);
+    var initialIntervalStart = currentDate;
+    while (initialIntervalStart.getDate() != 1) {
```

Comment:
Is this just looking for the start of the month? date-fns startOfMonth

Reply:
Ah good call, copy/paste flog